src/components/MessagesList/MessagesGroup/MessagesGroup.vue

```diff
@@ -31,7 +31,7 @@
 					:author-id="actorId"
 					:display-name="actorDisplayName" />
 			</div>
-			<div class="messages">
+			<ul class="messages">
```

Comment:
The list is `ol` though, as it’s ordered, no?

Reply:
indeed, had the same thought.
technically doesn't make a difference for assistive technologies from what I read

Reply:
The only difference afaik is visual in default styles. Bullets in <ul> and umbers in <ol>

Reply:
Also when scrolling up new items would be added and therefor all numbers would change. sounds confusing?